package.json

```diff
@@ -21,10 +21,11 @@
     "lint": "dtslint types"
   },
   "devDependencies": {
-    "dtslint": "Microsoft/dtslint#production",
+    "dtslint": "github:Microsoft/dtslint#production",
     "types-publisher": "Microsoft/types-publisher#production"
   },
   "dependencies": {
-    "jslint": "^0.10.3"
+    "jslint": "^0.10.3",
+    "typescript": "^2.4.2"
   }
```

Comment:
Any changes to the global `package.json` should go in their own PRs. I created #18746 to undo the addition of `jslint`.